sources/Interop/Windows/um/d3d10shader/Windows.Manual.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/d3d10shader.h in the Windows SDK for Windows 10.0.19041.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+namespace TerraFX.Interop
+{
+    public static unsafe partial class Windows
+    {
+        public static int D3D10_TX_VERSION(int _Major,int _Minor)
```

Comment:
```suggestion
        public static int D3D10_TX_VERSION(int _Major, int _Minor)
```

Reply:
Is this inlined under normal conditions? I've been putting `AggressiveInlining` on the `#define` ports, just to help ensure they are